UI/common-settings.cpp

```diff
@@ -0,0 +1,183 @@
+#include "common-settings.hpp"
+
+#include "audio-encoders.hpp"
+
+bool IsAdvancedMode(config_t *config)
```

Comment:
"static" is a good idea; avoids potential future conflicts in the global namespace.